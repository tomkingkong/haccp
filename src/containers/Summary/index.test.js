import React from 'react';
import { shallow } from 'enzyme';

import { Summary } from '.';

describe('Summary', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Summary />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});