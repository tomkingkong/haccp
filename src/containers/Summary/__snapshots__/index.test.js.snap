// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Summary should match snapshot 1`] = `
<div>
  <h2>
    Summary
  </h2>
  <button>
    Print
  </button>
</div>
`;
